include/ck/tensor_operation/gpu/device/impl/device_grouped_gemm_multiple_d_dl.hpp

```diff
@@ -607,8 +607,10 @@ struct DeviceGroupedGemmMultipleD_Dl : public DeviceGroupedGemm<ALayout,
                 }
             }
 
-            if(cpy_stream && cpy_event) // Custom flow, user allocates pinned memory for fully async
-                                        // host args copy
+            if(cpy_stream &&
+               cpy_event) // If user provides copy stream and copy event, we assume that he's also
+                          // responsible for providing allocated host memory (eg. pinned) which
+                          // would be used to copy kernel arguments to the device.
```

Comment:
Just my preference: comments above the line are more readable to me and you get more space.